### PR TITLE
feat: monitoring

### PR DESCRIPTION
**Pull request by v-rocheleau**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@ant-design/icons": "^5.3.0",
     "@ant-design/pro-components": "^2.7.0",
+    "@nicholasadamou/react-iframe": "^1.0.4",
```
Comment (davidlougheed) [inline package.json:9]: this package is old, and doesn't have peer dependencies that specify the correct react version - and it doesn't seem to be used anywhere?

File: src/components/App.js [hunk 2]
```diff
@@ -198,7 +199,7 @@ const App = () => {
 
     // On the cBioPortal tab only, eliminate the margin around the content
```
Comment (davidlougheed) [inline src/components/App.js:200]: update this comment to match the new logic

File: src/components/GrafanaContent.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, {useEffect} from "react";
```
Comment (davidlougheed) [inline src/components/GrafanaContent.js:1]: this file needs `prettier` ran on it - the linting doesn't seem to match the rest of the app

File: src/config.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ export const BENTO_CBIOPORTAL_PUBLIC_URL = BENTO_WEB_CONFIG.BENTO_CBIOPORTAL_PUB
     process.env.BENTO_CBIOPORTAL_PUBLIC_URL ?? null;
 export const CUSTOM_HEADER = BENTO_WEB_CONFIG.CUSTOM_HEADER ?? process.env.CUSTOM_HEADER ?? null;
 
+export const BENTO_GRAFANA_URL = `${BENTO_URL_NO_TRAILING_SLASH}/api/grafana`
```
Comment (davidlougheed) [inline src/config.js:14]: space this from the const below it with an extra newline

**Review by davidlougheed — CHANGES_REQUESTED** [review]

**Comment by davidlougheed** [comment]
forgot to mention but: also need to fix merge conflicts.

File: package.json (hunk 1 above)
Comment (CowTrainer) [reply package.json]: removed

File: src/components/SiteHeader.js [hunk 5]
```diff
@@ -17,9 +17,10 @@ import {
   PieChartOutlined,
   SettingOutlined,
   UserOutlined,
+  ApartmentOutlined,
```
Comment (davidlougheed) [inline src/components/SiteHeader.js:20]: alphabetical order with other icons

File: src/components/SiteHeader.js [hunk 6]
```diff
@@ -17,9 +17,10 @@ import {
...
 } from "@ant-design/icons";
 
-import { BENTO_CBIOPORTAL_ENABLED, CUSTOM_HEADER } from "@/config";
+import { BENTO_CBIOPORTAL_ENABLED, CUSTOM_HEADER, BENTO_MONITORING_ENABLED } from "@/config";
```
Comment (davidlougheed) [inline src/components/SiteHeader.js:23]: alphabetical order (before custom header)

File: src/components/GrafanaContent.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react";
```
Comment (davidlougheed) [inline src/components/GrafanaContent.js:1]: importing React here isn't needed anymore (see https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html )

**Review by davidlougheed — CHANGES_REQUESTED** [review]
missing property addition in static/config.js: https://github.com/bento-platform/bento_web/blob/master/static/config.js

**Review by davidlougheed — APPROVED** [review]
lgtm!

**Review by davidlougheed — APPROVED** [review]
lgtm. we'll need to eventually enforce permissions on this.

**Pull request merged**